x-pack/plugins/aiops/public/hooks/use_aiops_app_context.ts

```diff
@@ -115,6 +120,8 @@ export interface AiopsAppDependencies {
   embeddable?: EmbeddableStart;
   cases?: CasesUiStart;
   isServerless?: boolean;
+  /** Identifier to indicate the plugin utilizing the component */
+  embeddingOrigin: string;
```

Comment:
I find this naming a bit confusing, because it's not about embedding origin, but the app context in general. Lens embeddable uses `executionContext` name and I reckon we can do the same. You can check the example [here](https://github.com/elastic/kibana/blob/fb6ac2e4459ff5b981c41cdf410c2b321614942b/x-pack/plugins/aiops/public/components/change_point_detection/chart_component.tsx#L56).
```suggestion
  executionContext: { type: string; name: string }
```

Reply:
Just to clarify, my suggestion is _only_ about the naming for this property in the AiopsContext. For ui counter payload `embeddingOrigin` makes sense 👍🏻